runtime/gc_verbose_handler_vlhgc/VerboseHandlerOutputVLHGC.cpp

```diff
@@ -427,7 +427,8 @@ MM_VerboseHandlerOutputVLHGC::handleCopyForwardEnd(J9HookInterface** hook, UDATA
 		writer->formatAndOutput(env, 1, "<warning details=\"scan cache overflow (storage acquired from heap)\" />");
 	}
 	if (0 != copyForwardStats->_nonEvacuateRegionCount) {
-		writer->formatAndOutput(env, 1, "<warning details=\"running under hybrid mode, JNI Critical region count=%zu\" />", copyForwardStats->_nonEvacuateRegionCount);
+		writer->formatAndOutput(env, 1, "<warning details=\"running under hybrid mode, Non Evacurated region count=%zu, scanObjects=\"%zu\" markobjects=\"%zu\" scanBytes=\"%zu\" />",
```

Comment:
I would not report this as warning. Besides, we already report memory-trace stanza just a few lines above - could we not piggy back on it (for scan/mark objects)?
But yes, it would be nice to report additionally, how many regions were not subject to evacuation (in a separate line).

Reply:
there is spelling error here (and at several other spots - please search): Evacurate